.code-samples.meilisearch.yaml

```diff
@@ -0,0 +1,866 @@
+      switch result {
+      case .success(let update):
+          print(update)
+      case .failure(let error):
+          print(error)
+      }
+  }
+add_or_update_documents_1: |-
+  let document: Data = //...
+  client.addOrUpdateDocument(UID: "movies", document: document) { (result: Result<Update, Swift.Error>) in
+      switch result {
+      case .success(let update):
+          print(update)
+      case .failure(let error):
+          print(error)
+      }
+  }
+delete_all_documents_1: |-
+  client.deleteAllDocuments(UID: "movies") { (result: Result<Update, Swift.Error>) in
+      switch result {
+      case .success(let update):
+          print(update)
+      case .failure(let error):
+          print(error)
+      }
+  }
+delete_one_document_1: |-
+  client.deleteDocument(UID: "movies", identifier: 25684) { (result: Result<Update, Swift.Error>) in
+      switch result {
+      case .success(let update):
+          print(update)
+      case .failure(let error):
+          print(error)
+      }
+  }
+delete_documents_1: |-
+  client.deleteBatchDocuments(
+    UID: "movies", 
+    documentsUID: [23488, 153738, 437035, 363869]) { (result: Result<Update, Swift.Error>) in
+        switch result {
+        case .success(let update):
+            print(update)
+        case .failure(let error):
+            print(error)
+        }
+    }
+search_1: |-
+  let searchParameters = SearchParameters.query("American ninja")
+  client.search(UID: "movies", searchParameters) { (result: Result<SearchResult<Movie>, Swift.Error>) in
+      switch result {
+      case .success(let searchResult):
+          print(searchResult)
+      case .failure(let error):
+          print(error)
+      }
+  }
+get_update_1: |-
+  client.getUpdate(UID: "movies", 1) { (result: Result<Update.Result, Swift.Error>) in
+      switch result {
+      case .success(let update):
+          print(update)
+      case .failure(let error):
+          print(error)
+      }
+  }
+get_all_updates_1: |-
+  client.getAllUpdates(UID: "movies") { (result: Result<[Update.Result], Swift.Error>) in
+      switch result {
+      case .success(let updates):
+          print(updates)
+      case .failure(let error):
+          print(error)
+      }
+  }
+get_keys_1: |-
+    let masterKey = "c4dff5a196824a0704c2023916c21eaf0a13485a9d637416820bc623375f2a"
+    client.keys(masterKey: masterKey) { (result: Result<Key, Swift.Error>) in
+        switch result {
+        case .success(let key):
+            print(key)
+        case .failure(let error):
+            print(error)
+        }
+    }
+get_settings_1: |-
+  client.getSetting(UID: "movies") { (result: Result<Setting, Swift.Error>) in
+      switch result {
+      case .success(let setting):
+          print(setting)
+      case .failure(let error):
+          print(error)
+      }
+  }
+update_settings_1: |-
+  let settings = Settings(...)
```

Comment:
Can you add the same example https://docs.meilisearch.com/references/settings.html#example-2 ? 🙂